assets/src/pages/room/hooks/usePinning.tsx

```diff
@@ -36,7 +40,14 @@ const usePinning = (): PinningApi => {
     }
   };
 
-  return { pinnedTileIds, pin, unpin, pinIfNotAlreadyPinned };
+  const removePinnedEarlier = (localTileId: string, remoteTileId: string) => {
+    if (!lastPinnedTileId) return;
+
+    if (localTileId === lastPinnedTileId) unpin(remoteTileId);
+    if (remoteTileId === lastPinnedTileId) unpin(localTileId);
+  };
+
+  return { pinnedTileIds, pin, unpin, pinIfNotAlreadyPinned, removePinnedEarlier };
```

Comment:
usePinnig hook now knows, that there are local and remote tiles. I think is unnecessary

Reply:
This method was removed